pkgs/applications/networking/instant-messengers/riot/update-riot-desktop.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -I nixpkgs=../../../../../ -i bash -p wget yarn2nix-moretea.yarn2nix
+
+ set -euo pipefail
+
+ if [ "$#" -ne 1 ] || [[ "$1" == -* ]]; then
+  echo "Regenerates the Yarn dependency lock files for the riot-desktop package."
+  echo "Usage: $0 <git release tag>"
+  exit 1
+fi
+
+ RIOT_WEB_SRC="https://raw.githubusercontent.com/vector-im/riot-web/$1"
+
+ wget "$RIOT_WEB_SRC/electron_app/package.json" -O riot-desktop-package.json
```

Comment:
Nitpick: weird indent at lines 4, 6, 12 and 14.

Reply:
Fail while pasting, thanks for noticing